src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -7384,6 +7384,7 @@ private Task WriteEncodingChar(string s, Encoding encoding, TdsParserStateObject
 
         private byte[] SerializeEncodingChar(string s, int numChars, int offset, Encoding encoding)
         {
+#if NETFRAMEWORK || NETSTANDARD2_0
             char[] charData;
```

Comment:
You can remove the NETFRAMEWORK since this is the netcore file. Same applies to the NETSTANDARD2_0 in the netfx changes.

Reply:
I did that so the code in both files would be identical so as to aid in merging the 2 codebases.